index.scss

```diff
@@ -137,27 +137,61 @@ $breakpoints-ie8: false!default;
   @return $value;
 }
 
+@function _set-max-breakpoint($max-breakpoint) {
+  @if $max-breakpoint != false {
+    @if unit($max-breakpoint) == px {
+      $max-breakpoint: $max-breakpoint - 1;
```

Comment:
exactly.

Initially I wanted to use `calc(any-type-of-unit - 1px)` but media queries do not work with `calc` 👎  so this is addressing the issue where, if you're using `rem`, you don't get a `1rem` gap where no breakpoints are applied.